src/test/java/com/philips/research/bombar/core/domain/ProjectInteractorTest.java

```diff
@@ -210,6 +208,26 @@ void readsLicenseDistributionForProject() {
             assertThat(distribution).containsEntry("B", 1);
         }
 
+        @Test
+        void noObligationsForEmptyProject() {
+            project.clearDependencies();
+
+            assertThat(interactor.findObligations(PROJECT_ID)).isEmpty();
+        }
+
+        @Test
+        void readObligationsForProject() {
+            final var analyzer = mock(ObligationsAnalyzer.class);
+            final var obligation = "Obligation";
```

Comment:
This looks like a candidate for extraction as an `OBLIGATION` constant.

Reply:
this constant is used only for this test, why do you want to extract?

Reply:
To me this one sounds sufficiently generic to promote to a constant. But I admit this is a matter of personal taste.

Reply:
I keep it same.